src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -53,4 +95,34 @@ private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Pre
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
 
+    /**
+     * Throws ParseException when one of the required prefixes for {@code Patient} are absent.
+     */
+    private static void arePrefixesPresentPatient(ArgumentMultimap argMultimap) throws ParseException {
+        if (!arePrefixesPresent(argMultimap, PREFIX_TYPE, PREFIX_NRIC, PREFIX_AGE, PREFIX_NAME, PREFIX_PHONE)
+                || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
+        }
+    }
+
+    /**
+     * Throws ParseException when one of the required prefixes for Donor are absent.
+     */
+    private static void arePrefixesPresentDonor(ArgumentMultimap argMultimap) throws ParseException {
+        if (!arePrefixesPresent(argMultimap, PREFIX_TYPE, PREFIX_NRIC, PREFIX_AGE, PREFIX_NAME, PREFIX_PHONE)
```

Comment:
Same here

Reply:
Rectified in commit "Improve coding style of AddCommandParser.java"